concepts/eventing/performance evaluation/nats-jetstream/results_prod.md

```diff
@@ -23,9 +22,15 @@
   ```
   kyma deploy --source=main -p production --value global.jetstream.enabled=true --value global.jetstream.storage=file
   ```
+* Testing tool: [K6](https://k6.io/) (deployed in the same Kyma cluster)
+  * Grafana Dashboard:
+    * **Avg k6-to-eventing proxy time:** Shows the average time taken by the event to be sent from the sender till its received by the event-publisher-proxy.
+    * **Avg e2e time:** Shows the average end-to-end time taken by the event to be sent from the sender till its received by the final sink (i.e. the receiver).
 
 ## Test Scenario 1: Without any server restarts/crash
 
+In this test scenario, the normal behaviour of NATS was tested without any server restarts or crash. The goal of was to check that there is no event loss in case of high event throughput.
```

Comment:
```suggestion
In this test scenario, the normal behaviour of NATS was tested without any server restarts or crash. The goal was to check that there is no event loss in case of high event throughput.
```